app/models/article.rb

```diff
@@ -930,7 +925,7 @@ def set_nth_published_at
   end
 
   def title_to_slug
-    "#{Sterile.sluggerize(title)}-#{rand(100_000).to_s(26)}"
+    "#{Sterile.sluggerize(title)}-#{rand(100_000).to_s(26)}" # rubocop:disable Rails/ToSWithArgument
```

Comment:
Rubocop was complaining about this use of `to_s(Integer)` ([method docs](https://ruby-doc.org/core-3.0.2/Integer.html#method-i-to_s)) and suggesting to use `to_formatted_s/to_fs` instead ([method docs](https://api.rubyonrails.org/classes/ActiveSupport/NumericWithFormat.html#method-i-to_fs)).

The problem with the Rubocop suggestion is that `to_s(Integer)` transforms the integer into a string representation on the parameter's radix base (between 2 and 36), and the docs for `to_fs` don't seem to mention this is possible. A quick test locally was unsuccessful but I'm willing to make the change if anyone thinks it should be done regardless.